source/init/mus_initFluid_module.f90

```diff
@@ -277,17 +217,136 @@ subroutine mus_init_advRel_fluid( relaxation, layout, compute )
+      case default
+        call tem_abort('Stencil '//trim(layout)//' is not supported yet!')
+      end select
+
+    case default
+      call tem_abort('Unsupported variant "'//trim(variant)//'" for'// &
+        &             ' "bgk" relaxation!')
+    end select
+
+  end subroutine mus_init_advRel_fluid_bgk
+  ! ************************************************************************** !
+
+  ! ************************************************************************** !
+  !> This routine assigns compute routine for mrt relaxation
+  subroutine mus_init_advRel_fluid_mrt( variant, layout, compute )
+    ! --------------------------------------------------------------------------
+    character(len=labelLen), intent(in) :: variant
+    character(len=labelLen), intent(in) :: layout
+    procedure( kernel ), pointer, intent(out) :: compute
+    ! --------------------------------------------------------------------------
+    select case (trim(variant))
+    case ('standard')
+      select case (trim(layout))
+      case ('d3q27')
+        compute => mus_advRel_kFluid_rMRT_vStd_lD3Q27
+      case ('d3q19')
+        compute => mus_advRel_kFluid_rMRT_vStd_lD3Q19
+      case ('d2q9')
+        compute => mus_advRel_kFluid_rMRT_vStd_lD2Q9
+      case default
+        call tem_abort('Unsupported variant "'//trim(variant)//'" for'// &
+          &            ' "mrt" relaxation!')
+      end select
+
+    case ('standard_no_opt')
+      select case (trim(layout))
+      case ('d3q27')
+        compute => mus_advRel_kCFD_rMRT_vStdNoOpt_lD3Q27
+      case ('d3q19')
+        compute => mus_advRel_kFluid_rMRT_vStdNoOpt_lD3Q19
+      case default
+        compute => mus_advRel_kCFD_rMRT_vStdNoOpt_l
+      end select
+
+
+    case ('bgk')
+      compute => mus_advRel_kCFD_rMRT_vStdNoOpt_l
+
+    case default
+      call tem_abort('Unsupported variant "'//trim(variant)//'" for'// &
+        &            ' "mrt" relaxation!')
+    end select
+
+  end subroutine mus_init_advRel_fluid_mrt
+  ! ************************************************************************** !
+
+  ! ************************************************************************** !
+  !> This routine assigns compute routine for trt relaxation
+  subroutine mus_init_advRel_fluid_trt( variant, layout, compute )
+    ! --------------------------------------------------------------------------
+    character(len=labelLen), intent(in) :: variant
+    character(len=labelLen), intent(in) :: layout
+    procedure( kernel ), pointer, intent(out) :: compute
+    ! --------------------------------------------------------------------------
+    select case (trim(variant))
+    case ('standard')
+      select case (trim(layout))
+      case ('d3q19')
+        compute => mus_advRel_kFluid_rTRT_vStd_lD3Q19
+      case ('d3q27')
+        compute => mus_advRel_kFluid_rTRT_vStd_lD3Q27
+      case default
+        call tem_abort('Stencil '//trim(layout)//' is not supported yet!')
```

Comment:
I'd prefer to refer to the name we use in the UI, which to my understanding is `layout`, instead of `stencil`.